Library/Homebrew/cmd/outdated.rb

```diff
@@ -78,6 +78,7 @@ def run
           end
 
           print_outdated(outdated)
+          print_disabled(disabled_casks)
```

Comment:
As mentioned: if we output casks here: we should also output formulae. May also want to think about how `--json` is handled.